charts/cronjob-ldap-group-sync/templates/cronjob.yaml

```diff
@@ -39,5 +39,5 @@ spec:
           - name: ldap-bind-password
             secret:
               secretName: {{ .Values.ldap_bind_password_secret }}
-  schedule: {{ .Values.schedule }}
+  schedule: '{{ .Values.schedule }}'
```

Comment:
What do you think of `{{ .Values.schedule | quote }}` to match with upstream k8s[0] and use a `"` string?

[0] https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#example

Reply:
haha, of course, that'd be way better 🙈 Thanks!